README.md

```diff
@@ -25,6 +25,30 @@ either in France or Hong Kong.
 
 ### Usage
 
+```bash
```

Comment:
This help is already available in the program so I prefer to not duplicate the code.